packages/@webex/http-core/src/lib/detect.js

```diff
@@ -3,6 +3,7 @@
  */
 
 import {fromBuffer} from 'file-type';
+import {Blob} from 'node:buffer';
```

Comment:
We do not want to import Blob from `node:buffer`. This code runs in the browser and this will not be available, nor is it necessary since Blob is globally available in the browser